tests/tests/client-struct-unmount-race.sh

```diff
@@ -0,0 +1,66 @@
+#
+# Unmount Server and unmount a client as it's replaying to a remaining server
+#
+
+t_require_mounts 3
+majority_nr=$(t_majority_count)
+quorum_nr=$T_QUORUM
+
+cur_quorum=$quorum_nr
+test "$cur_quorum" == "$majority_nr" && \
+        t_skip "all quorum members make up majority, need more mounts to unmount"
+
+test "$T_NR_MOUNTS" -lt "$T_QUORUM" && \
+        t_skip "Need enough non-quorum clients to unmount"
+
+echo "== create per mount files"
```

Comment:
These "== " echos are meant to leave breadcrumbs in the logs that indicate what was happening when a test failed.   So this one that describes something that isn't happening should be removed, and the next two should be combined into the single one that describes what's happening: LENGTHs of unmounting non-quorum clients during recovery.